workflows/process/collect_fastq.nf

```diff
@@ -5,16 +5,28 @@ process collect_fastq {
     output:
         tuple val(name), path("*.fastq.gz"), emit: reads
     script:
+        if (params.rapid) {
+            guppy_arrangement_files = 'barcode_arrs_rbk4.cfg'
```

Comment:
@DataSpott this needs the rbk file that would support 96 barcodes (check newest guppy)